app/services/upload_pdf.rb

```diff
@@ -0,0 +1,23 @@
+# frozen_string_literal: true
+
+require 'http'
+
+module CoEditPDF
+  # Creates an account
+  class UploadPdf
+    class UploadError < StandardError; end
+
+    def initialize(config)
+      @config = config
+    end
+
+    def call(current_account:, filename:, file_read:)
```

Comment:
I think "content" or "file_content" would be less confusing than "file_read".

Reply:
fixed